src/core/tests/CMakeLists.txt

```diff
@@ -427,6 +427,7 @@ set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/threading.cpp
 
 if(SUGGEST_OVERRIDE_SUPPORTED)
     set_source_files_properties(ov_tensor_test.cpp
+                                type_prop/multiclass_nms.cpp
```

Comment:
Better to remove it.

Reply:
We have to. Confirmed with Ilya before.

Reply:
gtest has an issue when build with c++11 and "-Wsuggest-override" . This option would lead to failure when use "TYPED_TEST" for this component. https://github.com/google/googletest/issues/1063